projects/plugins/boost/app/modules/image-guide/Image_Guide_Proxy.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Modules\Image_Guide;
+
+/**
+ * Add an ajax endpoint to proxy external CSS files.
+ */
+class Image_Guide_Proxy {
+	const NONCE_ACTION = 'jb-ig-proxy-nonce';
+
+	public static function init() {
+		$instance = new self();
+		add_action( 'wp_ajax_boost_proxy_ig', array( $instance, 'handle_ig_proxy' ) );
```

Comment:
Why instantiate at all? Seems like we can just make `handle_ig_proxy` static and avoid instantiating :)

Reply:
Good idea, done!

Reply:
Nice. Thank you :) 

This is a totally optional thing, but did you know you can avoid having to spell out the full namespace in the hook name by using `__NAMESPACE__` like this example:

https://github.com/Automattic/jetpack/blob/3c05799e6811f48b1352a1cd38a669272e4a92cf/projects/plugins/boost/jetpack-boost.php#L149

It's a trick I like to use.